frontend/.storybook/preview.js

```diff
@@ -32,13 +32,11 @@ export const decorators = [
       }}
     >
       <Provider store={store}>
-        <Router>
```

Comment:
The reason being... No test will want to navigate?

Reply:
i removed it because you cant have a MemoryRouter under a Router. And i needed a memory router defined story level because it takes a specific route structure. I will rewrite it so its more global